sycl/test/abi/layout_buffer.cpp

```diff
@@ -0,0 +1,96 @@
+// RUN: %clangxx -fsycl -c -fno-color-diagnostics -Xclang -fdump-record-layouts %s | FileCheck %s
+// REQUIRES: linux
+
+// clang-format off
+
+#include <CL/sycl/buffer.hpp>
+
+void foo(sycl::buffer<int, 2>) {}
+// CHECK: 0 | class cl::sycl::detail::buffer_impl
+// CHEK-NEXT: 0 |   class cl::sycl::detail::SYCLMemObjT (primary base)
```

Comment:
```suggestion
// CHECK-NEXT: 0 |   class cl::sycl::detail::SYCLMemObjT (primary base)
```
Applies to all lines